src/WebJobs.Extensions.DurableTask/LocalGrpcListener.cs

```diff
@@ -217,7 +217,7 @@ public override Task<Empty> Hello(Empty request, ServerCallContext context)
                     // Create the orchestration instance
                     var instance = new OrchestrationInstance
                     {
-                        InstanceId = request.InstanceId ?? Guid.NewGuid().ToString("N"),
+                        InstanceId = string.IsNullOrEmpty(request.InstanceId) ? Guid.NewGuid().ToString("N") : request.InstanceId,
```

Comment:
Does this issue also apply to .NET Isolated and Java? Since I remember we already assign the value for InstanceId at isolated sdks

Reply:
I didn't find the bug when testing with .NET Isolated or Java, but I just added the check here so it's consistent.